parser/clj_kondo/impl/rewrite_clj/parser/utils.clj

```diff
@@ -22,13 +22,19 @@
   (r/read-char reader)
   nil)
 
+(defn ex-finding [row col message cause-ex]
+  (ex-info message {:type :syntax
+                    :col col
+                    :row row
+                    :message (clojure.string/trim message)}
+           cause-ex))
+
 (defn throw-reader
   [reader & msg]
```

Comment:
So it turns out that `rewrite-_clj/parser` has two `throw-reader` functions, with slightly different signatures. I make one of them call the other one in this change. It would be better to make this change in an isolated PR.

Or maybe we want to minimize changes to `rewrite-clj` and we can make the `ex-data` change in two places.